src/client/components/SingleMovie/SingleMovie.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Logo from '../Logo';
+import './style.scss';
+
+export default class SingleMovie extends PureComponent {
+  static propTypes = {
+    movieId: PropTypes.number.isRequired,
+    handleClickToSearch: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      movie: {},
+    };
+  }
+
+  componentDidMount() {
+    const { movieId } = this.props;
+    const callString = `http://react-cdp-api.herokuapp.com/movies/${movieId}`;
+
+    fetch(callString)
+      .then(response => response.json())
+      .then((myJson) => {
+        this.setState({
+          movie: myJson,
+        });
+      });
+  }
+
+  handleSearchButton() {
+    const { handleClickToSearch } = this.props;
+    handleClickToSearch.call();
+  }
+
+  render() {
+    const { movie } = this.state;
+
+    return (
+      <div className="single-movie-wrapper">
+        <Logo />
+        <img src={movie.poster_path} alt={movie.title} />
+        <button className="btn-search" type="button" onClick={this.handleSearchButton.bind(this)}>search</button>
```

Comment:
The same as above, no binding in render method.

Reply:
done